méli-mélo/2024-10-datatable-utilities/index-en.html

```diff
@@ -0,0 +1,319 @@
+---
+title: DataTable Utilities
+dateModified: 2024-10-31
+description: "Utilities that allow Columns to be MONEY Formatted, Emails wrapped with mailto, url's to be clickable and Footer totals that can be affected by filtered Data Or not."
+lang: en
+altLangPage: "index-fr.html"
+script: [ "js/wb-pspc-datatable-utility.js" ]
+css: [ "css/wb-pspc-datatable-utility.css"]
+---
+<p>DataTable Utilities, includes Data Manipulation Classes for emails, Urls and Money. Also Includes Datatable Footer Totals.</p>
+<p>Having this plugin intergrated will make the transition to Canada.ca easier as these features are already in use on https://www.tpsgc-pwgsc.gc.ca, currently some of the pages that make use of this plugin and other pages that make use of Other Javascripts are just pointed to from Canada.ca</p>
+<p>The Public at Large, the plugin makes DataTables more user friendly with Clickable Links, and they are used to this functionality our PRE Canada.ca Pages.</p>
+
+<h2>GCWeb implementation plan</h2>
+
+{% assign implPlan = site.pages | where: "output", "false" | where: "componentName", "2024-10-datatable-utilities" | first %}
+
+<ul>
+    {% for deliverable in implPlan.implementationPlan %}
+    <li>{{ deliverable.due }} - {{ deliverable.what }}</li>
+    {% endfor %}
+</ul>
+
+<p>Todo and for future consideration</p>
+<ul>
+    {% for todo in implPlan.todos %}
+    <li>{{ todo }}</li>
+    {% endfor %}
+</ul>
+
+<p>Sponsor: PSPC - {{ implPlan.sponsor }}</p>
```

Comment:
Since there is a documentation page now. Could you please move this to the documentation page?